sdk/genesis-config/Cargo.toml

```
@@ -0,0 +1,47 @@
+[package]
+name = "solana-genesis-config"
+description = "The chain's genesis config."
```

nit: wanna call out that it's for solana?
```suggestion
description = "A Solana network's genesis config."
```